templates/core/terraform/main.tf

```diff
@@ -14,7 +14,7 @@ provider "azurerm" {
   features {
     key_vault {
       purge_soft_delete_on_destroy    = var.keyvault_purge_protection_enabled ? false : true
-      recover_soft_deleted_key_vaults = true
+      recover_soft_deleted_key_vaults = false
```

Comment:
Was this intended to be included in this PR? It's not in the PR description (and probably warrants discussion?)

Reply:
It was an attempt to fix an env problem. Reverted now.